details.js

```diff
@@ -0,0 +1,150 @@
+                    <img class="desktop" src="${Articles[i].desktopImg}">
+                </div>
+                <div class="work_info">
+                    <h2>${Articles[i].heading}</h2>
+                    <div class="canopy">
+                        <div>
+                            <h5>CANOPY</h5>
+                        </div>
+                        <ul class="canopy_languages">
+                            <li>
+                                <img src="images/Counter.png" alt="counter image">
+                                <span>Backend dev</span>
+                            </li>
+                            <li>
+                                <img src="images/Counter.png" alt="counter image">
+                                <span>2015</span>
+                            </li>
+                        </ul>
+                    </div>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="languages">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <button class= "see" type="button">See Project</button>
+                </div>
+
+            </article>
+             <!--pop up-->
+        <div class="desktopPopup">
+            <div class="desktopPopup_header">
+                <h2>Tonic</h2>
+                <button type='button' id= "closePopup" class="closePopup">x</button>
+            </div>
+            <div class="canopy ">
+                <div>
+                    <h5>CANOPY</h5>
+                </div>
+                <ul class="canopy_languages">
+                    <li>
+                        <img class="smallImg" src="images/Counter.png" alt="counter image ">
+                        <span>Backend dev</span>
+                    </li>
+                    <li>
+                        <img class="smallImg" src="images/Counter.png" alt="counter image ">
+                        <span>2015</span>
+                    </li>
+                </ul>
+            </div>
+            <div class="popupImg">
+                <img src="${Articles[i].desktopImg}" alt="popup image">r
+            </div>
+            <div class="desktopPopup_descriptions">
+                <div>
+                    <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Provident, sequi fuga veniam labore
+                        ullam eos ducimus magni dolor facere saepe.</p>
+                </div>
+                <div>
+                    <ul class="languages">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <button class="popup-links" type="button">see live</button>
+                    <button class="popup-links" type="button">See source</button>
```

Comment:
- [ ] 👏 Good job adding buttons to your project! However, I noticed that the button styles don't quite match the design requirements. To improve this, I recommend following the design more closely and updating your code to include all button styles, including active and hover states. 

![Design Example](https://github.com/MPDADDY/my_portfolio/assets/55840999/6779d62d-e384-4796-9941-15718adb16a5)

- [ ] Additionally, the design includes an icon for each button, but I noticed that you haven't added the icons in your code. I recommend adding the proper icons as shown in the design.

Please pay attention to font styles, colors, and padding to ensure they align with the design. By making these adjustments, your buttons will look more visually appealing and consistent with the overall design. Keep up the good work! 👍🎨🚀